crates/cheatnet/src/execution/cheatable_syscall_handler.rs

```diff
@@ -44,7 +44,7 @@ pub struct CheatableSyscallHandler<'a> {
 }
 
 impl<'a> CheatableSyscallHandler<'a> {
-    pub fn new(
+    pub fn wrap(
```

Comment:
Can we rename it also for `TestExecutionSyscallHandler` and `ContractExecutionSyscallHandler`